webapp/src/hooks/customHooks/useIncomeReportState.js

```diff
@@ -193,6 +197,33 @@ const useIncomeReportState = () => {
       setTotalClaimedAndUnclaimedList(newFormatData)
     }
 
+  const newDataFormatPercentAllElections = percentAllElectionData => {
```

Comment:
we can omit the return if we don't do it anything more han model data

example: 
```
  const newDataFormatPercentAllElections = percentAllElectionData => {
    const newFormatData = percentAllElectionData.map(data => ({
      name: data.election,
      EOS_CLAIMED: Number(data.eos_claimed.toFixed(4)) * 100,
      EOS_UNCLAIMED: Number(data.eos_unclaimed.toFixed(4)) * 100,
      USD_CLAIMED: Number(data.usd_claimed.toFixed(4)) * 100,
      USD_UNCLAIMED: Number(data.usd_unclaimed.toFixed(4)) * 100
    }))

    setPercentIncomeList(newFormatData)
  }

  const newDataFormatPercentByElection = percentByElectionData => {
    const newFormatData = percentByElectionData.map(data => ({
      name: data.recipient,
      election: data.election,
      EOS_CLAIMED: Number(data.eos_claimed.toFixed(4)) * 100,
      EOS_UNCLAIMED: Number(data.eos_unclaimed.toFixed(4)) * 100,
      USD_CLAIMED: Number(data.usd_claimed.toFixed(4)) * 100,
      USD_UNCLAIMED: Number(data.usd_unclaimed.toFixed(4)) * 100
    }))
    
    setPercentIncomeList(newFormatData)
  }